torch/jit/__init__.py

```diff
@@ -900,10 +900,65 @@ def _try_compile_weak_script(fn):
         _jit_internal.compiled_weak_fns[fn]["compiled_fn"] = compiled_fn
         entry["status"] = _jit_internal.COMPILED
         return compiled_fn
+        # TODO: use fn.__closure__
+        raise RuntimeError("Cannot make resolutionCallback in Python 2")
     else:
         return entry["compiled_fn"]
 
 
+class ScriptWarning(Warning):
+    pass
+
+
+def make_rcb(fn):
```

Comment:
createResolutionCallback, make_rcb <- by name these appear to be the same thing. Can they be made more self explainatory when to use one or the other?